src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -658,7 +658,8 @@ FixedwingPositionControl::control_position(const hrt_abstime &now, const Vector2
 		_tecs.reset_state();
 	}
 
-	if ((_control_mode.flag_control_auto_enabled || _control_mode.flag_control_offboard_enabled) && pos_sp_curr.valid) {
+	if (((_control_mode.flag_control_auto_enabled && _control_mode.flag_control_position_enabled) ||
+	     _control_mode.flag_control_offboard_enabled) && pos_sp_curr.valid) {
```

Comment:
@sfuhrer Could it happen that we don't have a valid position setpoint and thus are not able to execute a fixed bank loiter?

Reply:
You're about the best one to answer that question;) I've so far not run into that, but I also haven't checked all the Navigator logic behind it. Would you know a reason why it could get invalid?